src/test/scala/fr/polytechnique/cmap/cnam/etl/extractors/acts/DcirBiologyActsSuite.scala

```diff
@@ -0,0 +1,270 @@
+package fr.polytechnique.cmap.cnam.etl.extractors.acts
+
+import fr.polytechnique.cmap.cnam.SharedContext
+import fr.polytechnique.cmap.cnam.etl.events.{BiologyAct, BiologyDcirAct, DcirAct, Event}
+import fr.polytechnique.cmap.cnam.etl.sources.Sources
+import fr.polytechnique.cmap.cnam.util.functions._
+import org.apache.spark.sql.catalyst.expressions.GenericRowWithSchema
+import org.apache.spark.sql.types._
+import org.scalatest.Matchers.{an, convertToAnyShouldWrapper}
+import org.scalatest.TryValues._
+
+import scala.util.Success
```

Comment:
Should be the highest on the import order.

Reply:
fixed